.github/workflows/detect_test_pollution.yml

```diff
@@ -43,7 +43,8 @@ jobs:
         -   name: Setup python
             uses: actions/setup-python@v4
             with:
-                python-version: 3.11
+                python-version: '3.12'
+                allow-prereleases: false
```

Comment:
set the pre-release to false, eventhough it is the default I feel it can be nice to keep it around for when we want to start checking new python pre-release